### ISSUE TITLE
Unable to select all settings in Deployment Plan Site Settings at once

### ISSUE THREAD

**Issue by JoshTango** [OrchardCMS/OrchardCore#12497]
I noticed while making a deployment plan recently that the site settings step does not have a checkbox for All.
I'm not sure if you want to add one or this is intentional.

**Comment by hishamco** [comment]
Please screenshot?

**Comment by JoshTango** [comment]
Am I the only one who sees it?

**Comment by hishamco** [comment]
Probably not ;)

**Comment by JoshTango** [comment]
hmmm i am using a default tenant installed 2 years ago but I did generate a brand new tenant to test in.

**Comment by hishamco** [comment]
Can you reproduce the issue after you upgrade to `v1.4`?

**Comment by JoshTango** [comment]
![image](https://user-images.githubusercontent.com/42816989/192897286-9d5eec3a-35ae-4d28-a2dc-b99355dc1645.png)

**Comment by JoshTango** [comment]
Ok so I attached a photo of my site settings step.  Now keep in mind I stalled my default tenant 2 years ago.  Maybe thats an issue.  I have upgraded to 1.4

**Comment by hishamco** [comment]
We need to check this before `v1.5`

**Comment by sebastienros** [comment]
Not blocking 1.5, why would it? Not a security issue, not a bug, just painful.

**Comment by hishamco** [comment]
According to my understanding there are no checkboxes to select the settings which is of course a bug

**Comment by JoshTango** [comment]
No.  There is no ALL checkbox.  You are forced to select them one by one.  Not a bug.

**Comment by hishamco** [comment]
Nah, I though you mean all the checboxes are missing, you are right Seb it could be moved into a backlog
